mitiq/shadows/shadows_utils.py

```diff
@@ -8,13 +8,74 @@
 # LICENSE file in the root directory of this source tree.
 
 """Defines utility functions for classical shadows protocol."""
-from typing import List, Tuple
+from typing import Tuple, List, Any
 
 import numpy as np
 from numpy.typing import NDArray
+import cirq
 
 import mitiq
 
+from itertools import product
+
+Paulis = [
+    cirq.I._unitary_(),
+    cirq.X._unitary_(),
+    cirq.Y._unitary_(),
+    cirq.Z._unitary_(),
+]
+
+
+def kronecker_product(matrices: List[NDArray[Any]]) -> NDArray[Any]:
+    """
+    Returns the Kronecker product of a list of matrices.
+    """
+    result = matrices[0]
+    for matrix in matrices[1:]:
+        result = np.kron(result, matrix)
+    return result
+
+
+# Input must be a 2x2 matrix, output is a vector of 4 elements
+def operator_ptm_vector_rep(opt: NDArray[Any]) -> NDArray[Any]:
+    r"""
+    Returns the PTM vector representation
+    :math:`|o\rangle\!\rangle\in \mathcal{H}_{4^n}`
+    of an operator :math:`opt\in \mathcal{L}(\mathcal{H}_{2^n})`.
```

Comment:
You'll have to double check this renders correctly, but $opt\in \mathcal{L}(\mathcal{H}_{2^n})$ is not what we want.
```suggestion
    of an operator :math:`\mathtt{opt}\in \mathcal{L}(\mathcal{H}_{2^n})`.
```

Reply:
Thanks, will change it fonts